reframe/core/pipeline.py

```diff
@@ -557,7 +557,7 @@ class RegressionTest(metaclass=RegressionTestMeta):
     #: :default: ``'10m'``
     #:
     #: .. note::
-    #:    .. versionchanged:: 2.15
+    #:    .. versionchanged:: 3.0
```

Comment:
No, the 3.0 changes should documented separately with a note on what has changed.